app/scripts/controllers/incoming-transactions.test.js

```diff
@@ -832,6 +842,97 @@ describe('IncomingTransactionsController', function () {
     });
   });
 
+  describe('block explorer lookup', function () {
+    let sandbox;
+
+    beforeEach(function () {
+      sandbox = sinon.createSandbox();
```

Comment:
This seems like a great solution! Though for future reference, this isn't really what the sandbox feature is for.  We shouldn't need to use `createSandbox` except in rare circumstances where we want independent sets of mocks.

`sinon` has  'default sandbox" that is used for all top-level calls (e.g. `sinon.stub`, `sinon.restore`, etc.) that we should be using in most cases. This avoids the hazards using a sandbox can bring. Using a non-default sandbox makes it too easy to forget to restore mocks (e.g. maybe the global `sinon.stub()` was used somewhere and there's no `sinon.restore()` to restore the global sandbox, letting that stub leak between tests)